app/assets/stylesheets/application.scss

```diff
@@ -30,6 +30,20 @@
  //   -ms-transform: translateY(-50%);
  //   transform: translateY(-50%);
  // }
+
+.jumbotran {
+    position: relative;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered background-color, background-size, border, border-radius, height, overflow, padding, position, width